build_tools/make_pkg.sh

```diff
@@ -23,19 +23,17 @@ set -e
 
 PKGDIR=`mktemp -d`
 
+SRC_DIR=$PWD
```

Comment:
There's some stuff further up in this file that can go - `make distclean` and possibly the version machinery (derive it from FBVF?)